source/includes/steps-install-shell-base.yaml

```
@@ -23,12 +23,18 @@ ref: extract-archive
 level: 4
 content: |
    
-   .. code-block::
+   .. code-block:: sh
 
       tar -zxvf path/to/archive
 
-   Skip this step if your web browser automatically unzips the file as
-   part of the download.
+   If your web browser automatically extracts the archive as
+   part of the download or you extract the archive without the
+   ``tar`` command, you may need to run the following command to
+   make ``mongosh`` executable:
+
+   .. code-block:: sh
+
+     chmod +x /path/to/mongosh
```

```suggestion
      chmod +x /path/to/mongosh
```
seems like indentation is off -- doesn't seem to affect the rendering in nextgen, however.

---

Updating.